### ISSUE TITLE
Web connectivity test contains null in blocking key 

### ISSUE THREAD

**Issue by lorenzoPrimi** [measurement-kit/measurement-kit#867]
When running the latest version of the NetProbe app (using mk 0.3.2) the web connectivity test contains a null key in the blocking and accessible key.

Runned with:

```
mk::ooni::WebConnectivity()
    .set_options("backend", "https://a.web-connectivity.th.ooni.io:4442")
    .set_options("nameserver", get_dns_server())
    .set_options("geoip_country_path", [geoip_country UTF8String])
    .set_options("geoip_asn_path", [geoip_asn UTF8String])
    .set_options("net/ca_bundle_path", [ca_cert UTF8String])
    .set_options("save_real_probe_ip", include_ip)
    .set_options("save_real_probe_asn", include_asn)
    .set_options("save_real_probe_cc", include_cc)
    .set_options("no_collector", !upload_results)
    .set_options("collector_base_url", [collector_address UTF8String])
    .set_input_filepath([path UTF8String])
    .set_output_filepath([[self getFileName:@"json"] UTF8String])
    .set_verbosity(MK_LOG_INFO)
```

urls.txt

```
http://torproject.org
http://ooni.nu
http://neubot.org
http://archive.org
http://creativecommons.org
http://cyber.law.harvard.edu
http://duckduckgo.com
http://netflix.com
http://nmap.org
http://www.emule.com
```

**Comment by bassosimone** [comment]
Thank you for reporting this bug! I've committed a hotfix in #871 and will merge it as soon as the automatic tests on travis-ci and circle-ci terminate.

**Comment by bassosimone** [comment]
Now #871 has been merged, so we can close this issue.

**Comment by hellais** [comment]
This is great! However I think that another fix is needed to also set the `blocking` key to false at the end of the if chain.

**Comment by bassosimone** [comment]
ack

**Comment by bassosimone** [comment]
Should be fixed in v0.3.5
